lib/chef/provider/file.rb

```diff
@@ -193,6 +199,12 @@ def manage_symlink_access?
 
       private
 
+      # Return true if we are running in a docker guest, or false otherwise.
+      def docker_guest?(node)
```

Comment:
(i don't know exactly what to do about back-compat with chef-sugar and how to deal with the way that chef-sugar has that method which takes a node as an argument or what that use case covers...)